### ISSUE TITLE
Any way to re-import an exported xliff?

### ISSUE THREAD

**Issue by jiangweiatgithub** [matecat/MateCat#3322]
If not, I guess that would be a great feature, as from time to time, we would prefer complete the mjaor translation/editing/proofreading off line.

**Comment by Ostico** [comment]
@jiangweiatgithub yes, you can export and re-import xliff files (by creating a new project). Matecat supports the trans-unit translation states to preserve your offilne work.
